Makefile

```diff
@@ -174,15 +174,16 @@ install-with-src-api: src/IdrisPaths.idr
 	${IDRIS2_BOOT} --install-with-src ${IDRIS2_LIB_IPKG}
 
 install-idris2:
-	mkdir -p ${PREFIX}/bin/
+	mkdir -p ${PREFIX}/bin/${NAME}_app
 	install ${TARGET} ${PREFIX}/bin
 ifeq ($(OS), windows)
 	-install ${TARGET}.cmd ${PREFIX}/bin
 endif
-	mkdir -p ${PREFIX}/lib/
-	install support/c/${IDRIS2_SUPPORT} ${PREFIX}/lib
```

Comment:
@gallais I'm rebasing, but saw this. This file/dir appears unused, removing it didn't break anything.

Regardless, shouldn't it be: `install ${TARGET}_app/${IDRIS2_SUPPORT} ${PREFIX}/lib`? (i.e. the same file it was compiled against, otherwise it could differ from the one in bin/idris2_app)

Reply:
Actually, shouldn't it be compiling bytecode against the freshly built lib instead of the previous version's copy? We can use `IDRIS2_LIBS` to make that happen.